endpoints/openrtb2/sample-requests/hooks/auction_bidder_reject.json

```diff
@@ -51,8 +51,7 @@
           {
             "id": "applogy-bid",
             "impid": "some-impression-id",
-            "price": 0,
-            "ext": {"prebid":{"type":"banner"}}
```

Comment:
Why did this `ext` need to be removed?

Reply:
It appears bid.ext was not validated in unit tests before.
Actual bid.ext turns out to be 
```
{"origbidcpm":0,"origbidcur":"USD","prebid":{"meta":{"adaptercode":"applogy"},"type":"banner"}}
```
 not 
```
"ext": {"prebid":{"type":"banner"}}
```

After I added bid.ext comparison logic it stoped working. 
File auction_test.go -> func assertBidResponseEqual: 
```
if len(expectedBid.Ext) > 0 {
	assert.JSONEq(t, string(expectedBid.Ext), string(actualBidMap[bidID].Ext), "Incorrect bid extension")
}
```
I removed bid.ext from expected result, because it was not validated before and it was not valid anyways. 
I can add expected bid.ext back and modify them accordingly. So far it's filled with default values only.